src/core/data-cite.js

```diff
@@ -52,11 +54,17 @@ function requestLookup(conf) {
     }
     switch (elem.localName) {
       case "a": {
+        if (elem.textContent === "") {
+          elem.textContent = title;
+        }
         elem.href = href;
         break;
       }
       case "dfn": {
-        wrapInner(elem, hyperHTML`<a href="${href}"></a>`);
+        const wrapper = hyperHTML`<a href="${href}">${
+          elem.textContent === "" ? title : ""
+        }</a>`;
+        wrapInner(elem, wrapper);
```

Comment:
@marcoscaceres, IMO this is weird as `wrapInner` is not intended to receive a wrapper with any children. It works anyway as we are not really doing `wrapInner` when `elem.textContent` is empty...

Reply:
True. Kept wondering about that. What do you suggest? Just use the element?

Reply:
How about:

```js
const anchor = hyperHTML`<a href="${href}">`;
if (!elem.textContent) {
  anchor.textContent = title;
  elem.append(anchor);
} else {
  wrapInner(elem, anchor);
}
```

Longer but IMO more readable without magic.

Reply:
Yeah, that looks better.

Reply:
Resolved as suggested by @saschanaz . :)